backend/worker/autoresolver.go

```diff
@@ -54,15 +54,19 @@ func (autoResolver *AutoResolver) resolveStaleErrorsForProject(ctx context.Conte
 
 	db := autoResolver.db
 
+	subQuery := db.
+		Model(model.ErrorObject{}).
+		Select("error_group_id").
+		Where("created_at >= ?", time.Now().AddDate(0, 0, -interval))
+
 	err := db.Debug().
```

Comment:
should we remove the `.Debug().` printing here for production?

Reply:
I don't think there's any harm in keeping it since there's nothing sensitive and that this will log every 30 minutes (current cron setup).